static/sass/_pattern_card.scss

```diff
@@ -55,7 +55,29 @@
     }
   }
 
+  .p-card--form {
```

Comment:
When naming patterns lets try and name them by the purposes. So they can be reused without odd context. I would suggest something like `.p-card--muted` or something.